drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -7,6 +7,12 @@
 
 #define DT_DRV_COMPAT nxp_lpc_spi
 
+#define PM_3_SUP 1
+
+#ifndef CONFIG_PM_POLICY_DEVICE_CONSTRAINTS
+#define CONFIG_PM_POLICY_DEVICE_CONSTRAINTS
+#endif
```

Comment:
this is very bad practice, do not do this, it is confusing. If the kconfig is not defined then it is not defined.